scripts/prettyjson

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
I also propose to call this `prettify_json`, since all others scripts so far use verbs...